src/Uno.UI.Toolkit/Storage/StorageFileHelper.cs

```diff
@@ -13,10 +13,10 @@ public partial class StorageFileHelper
 	/// </summary>
 	/// <param name="fileName">relative file path</param>
 	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
-	[NotImplemented("NET461")]
+
 	public static async Task<bool> ExistsInPackage(string fileName) => await FileExistsInPackage(fileName);
 
-#if NET461 || __NETSTD_REFERENCE__
+#if NET461 || __NETSTD_REFERENCE__ || WINDOWS || WINDOWS_UWP
```

Comment:
In this case, it would make sense to provide the UWP/WinAppSDK version, even it uses exceptions to provide the same feature.